layouts/shortcodes/asciinema.html

```diff
@@ -10,4 +10,4 @@
     {{ with .Get "columns" }}columns="{{.}}"{{ end }}>
 </asciinema-player>
 
-<script src="/js/asciinema-player.js"></script>
+<script src="js/asciinema-player.js"></script>
```

Comment:
This change makes it unusable on other pages then top level ones. I would recommend not doing it.